config/index.js

```diff
@@ -251,6 +251,22 @@ var conf = convict({
       format: String,
       default: undefined,
       env: 'SES_CONFIGURATION_SET'
+    },
+    bounces: {
+      hard: {
+        max: {
+          doc: 'Maximum number of hard bounces before blocking emails',
+          default: 0,
```

Comment:
If we keep `count > MAX`, then that would mean `0` is a valid max. I always find systems that overload their arguments to be confusing, and requires double checking each time I use one. I think a separate `enabled` property is more explicit.

I could add an `enabled` property, either for the whole bounce gating, or for each type.

```js
if (config.bounces.enabled) {

}
// or
if (config.bounces.hard.enabled) {

}
```